# Robot state
int32 state
int32 READY_FOR_ACTION = 0
int32 ACTION_RUNNING = 1
